source/tutorials/first-steps/ad-hoc-shell-environments.md

```diff
@@ -37,7 +37,7 @@ these 3 derivations will be built:
 
 Within the Nix shell, you can use the programs provided by these packages:
 
-```
+```console
```

Comment:
~~~suggestion
```shell-session
~~~

To match the surrounding code blocks

Reply:
Maybe single line commands need to be wrapped in "console"?